doc/source/whatsnew/v0.20.2.rst

```diff
@@ -1,7 +1,7 @@
 .. _whatsnew_0202:
 
-v0.20.2 (June 4, 2017)
-----------------------
+What's new in v0.20.2 (June 4, 2017)
+------------------------------------
```

Comment:
Are you sure you wanted to do this in `v.0.20.2` (which further increases the inconsistency) rather than `v.0.24.0`?

Reply:
Alternatively, you could add "What's new in " to all versions (which would be even better than introducing it starting with 0.23.0)